src/node/desktop/src/core/file-path.ts

```diff
@@ -43,6 +43,11 @@ function normalizeSeparatorsNative(path: string) {
   return normalizeSeparators(path, separator);
 }
 
+// Makes sure that the path only has `/` instead of `\` or `//`
+export const threatPathString = (path: string) => {
```

Comment:
I don't quote understand what this function is doing. It seems identical to https://github.com/rstudio/rstudio/pull/11168/files?w=1#diff-e763e5885fa9463dcc1490ef5c031c7eb737975eace1eecf59f4694e6ca82276R37-R39, except it doesn't let you choose the separator?

I also don't understand the name selected.

Reply:
I was afraid of messing with the `normalizeSeparators` or `normalizeSeparatorsNative`. 

In my tests for solving this particular issue, only removing any kind of `\\`, `\` over to `/` would work as expected. 

So I thought of using this standalone method. I think that we could even remove `normalizeSeparatorsNative` and use only `normalizeSeparators`, with the same Regexp you mentioned below. Altough I was afraid of breaking something and having to spend more time by testing out stuff. What are your thoughts on it?